### PR TITLE
Handle exception for key vault resource creation

### PR DESCRIPTION
**Pull request by Flanker32**
What does this implement/fix? Explain your changes.
---------------------------------------------------
…

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

**Review by wangmingliang-ms — APPROVED** [review]
LGTM

File: azure-toolkit-libs/azure-toolkit-keyvault-lib/src/main/java/com/microsoft/azure/toolkit/lib/keyvault/KeyVault.java [hunk 1]
```diff
@@ -147,5 +151,19 @@ public SecretAsyncClient getSecretClient() {
     public KeyAsyncClient getKeyClient() {
         return Optional.ofNullable(getRemote()).map(Vault::keyClient).orElse(null);
     }
+
+    @Nullable
+    public static Action<String> getAccessPolicyConfiureAction(@Nonnull KeyVault vault) {
+        final String url = vault.getPortalUrl() + "/access_configuration";
```
Comment (akaroml) [inline azure-toolkit-libs/azure-toolkit-keyvault-lib/src/main/java/com/microsoft/azure/toolkit/lib/keyvault/KeyVault.java:157]: Concatinating strings to construct a URL is not the right way. Please use URL class/functions instead.

File: azure-toolkit-libs/azure-toolkit-keyvault-lib/src/main/java/com/microsoft/azure/toolkit/lib/keyvault/secret/SecretVersionDraft.java [hunk 2]
```diff
@@ -47,16 +57,54 @@ public SecretProperties createResourceInAzure() {
...
+        try {
+            final String value = config.getValue();
+            final KeyVaultSecret secret = secretClient.setSecret(config.getName(), value).block();
+            final SecretProperties properties = Objects.requireNonNull(secret).getProperties();
+            Optional.ofNullable(config.getEnabled()).ifPresent(properties::setEnabled);
+            Optional.ofNullable(config.getContentType()).ifPresent(properties::setContentType);
+            return Objects.requireNonNull(secretClient.updateSecretProperties(properties).block());
+        } catch (final Throwable t) {
+            // swallow all exceptions to prevent credential leakage
+            final Action<String> configure = getAccessPolicyConfiureAction(keyVault);
```
Comment (akaroml) [inline azure-toolkit-libs/azure-toolkit-keyvault-lib/src/main/java/com/microsoft/azure/toolkit/lib/keyvault/secret/SecretVersionDraft.java:81]: Is it possible that the getAccessPolicyConfiureAction or getAccessPolicyLearnMoreAction throws?

**Review by akaroml — COMMENTED** [review]
Left some checkpoints.

**Pull request merged**